pyjamas-2019/videos/pyjamas2019-anti-patterns-oquesãoecomoselivrardelesempython-caiocarrara.json

```diff
@@ -0,0 +1,25 @@
+{
+    "copyright_text": null,
+    "description": "",
+    "duration": 3600,
+    "language": "por",
+    "recorded": "2019-12-14",
+    "related_urls": [
+        {
+            "label": "Conference schedule",
+            "url": "https://pyjamas.live"
+        }
+    ],
+    "speakers": [
+        "Caio Carrara"
+    ],
+    "tags": [],
+    "thumbnail_url": "https://github.com/pyjamasconf/pyjamas.live/blob/master/static/img/about/about.png",
+    "title": "[PyJamas-2019] Anti-patterns - o que são e como se livrar deles em Python - Caio Carrara",
```

Comment:
We usually remove the speaker name and conference name from the video title -- can you do this as well?